sympy/functions/special/error_functions.py

```diff
@@ -189,6 +206,17 @@ def _eval_rewrite_as_erfc(self, z):
     def _eval_rewrite_as_erfi(self, z):
         return -I*erfi(I*z)
 
+    def _eval_rewrite_as_Integral(self, z):
+        return self._as_integral(z, Dummy('t'))
```

Comment:
At first I thought this `_eval_rewrite_as_Integral` could now be:
````
    def _eval_rewrite_as_Integral(self, z):
        return self.func(z).as_integral()
````
But this makes me a bit nervous b/c I'm instantiating a new object whereas I think of these `_eval_rewrite_*` things as basically static class methods.  For example, things might get evaluated in some unwanted way, such as:
````
>>> erf(erfinv(z), evaluate=False)
erf(erfinv(z))
>>> _.rewrite(Integral, deep=False)
AttributeError: 'Symbol' object has no attribute 'as_integral'
````
(I think that should give an Integral with upper limit as `erfinv(z)`).

The `._as_intergral` method (again, basically static class method) is the only way I can see to do this without copy-pasting the integral **twice**.